sql-files/main.sql

```diff
@@ -286,6 +286,26 @@ CREATE TABLE IF NOT EXISTS `charlog` (
 --
 -- Table structure for table `elemental`
 --
+CREATE TABLE `clan` (
+  `clan_id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(24) NOT NULL DEFAULT '',
+  `master` varchar(24) NOT NULL DEFAULT '',
+  `mapname` varchar(24) NOT NULL DEFAULT '',
+  `max_member` smallint(6) unsigned NOT NULL DEFAULT '0',
+  PRIMARY KEY (`clan_id`)
+) ENGINE=MyISAM AUTO_INCREMENT=5 DEFAULT CHARSET=latin1;
+
+-- ----------------------------
+-- Records of clan
+-- ----------------------------
+INSERT INTO `clan` VALUES ('1', 'Swordman Clan', 'Raffam Oranpere', 'prontera', '500');
+INSERT INTO `clan` VALUES ('2', 'Arcwand Clan', 'Devon Aire', 'geffen', '500');
+INSERT INTO `clan` VALUES ('3', 'Golden Mace Clan', 'Berman Aire', 'prontera', '500');
+INSERT INTO `clan` VALUES ('4', 'Crossbow Clan', 'Shaam Rumi', 'payon', '500');
```

Comment:
Thanks seems like I forgot to add that update... >.<

I did not use constants because because the char server does not have a constant resolving feature in it yet. Any suggestions?